src/middlewared/middlewared/plugins/jail.py

```diff
@@ -989,6 +1048,103 @@ def rc_action(self, action):
+        system_ifaces = {i['name']: i for i in await self.middleware.call('interface.query')}
+        params = ['jail.query', [['OR', [['vnet', '=', 1], ['nat', '=', 1]]]]]
+        if await self.middleware.call('failover.status') == 'BACKUP':
+            params = ['failover.call_remote', params[0], [params[1]]]
+        for jail in (await self.middleware.call(*params) if not jails else jails):
+            nic = await self.middleware.call(
+                f'jail.retrieve_{"nat" if jail["nat"] else "vnet"}_interface',
+                jail['nat_interface' if jail['nat'] else 'vnet_default_interface']
+            )
+            if nic in system_ifaces:
+                conflicts = {'LRO', 'TSO4', 'TSO6', 'VLAN_HWTSO'} if jail['nat'] else {
+                    'TXCSUM', 'TXCSUM_IPV6', 'RXCSUM', 'RXCSUM_IPV6', 'TSO4', 'TSO6', 'LRO',
+                }
+                if not jail['nat']:
+                    bridges = await self.middleware.call('jail.retrieve_vnet_bridge', jail['interfaces'])
+                    if not bridges or not bridges[0]:
+                        continue
+                if not check_system_iface or set(system_ifaces[nic]['state']['capabilities']) & conflicts:
+                    jail_nics[nic].update(conflicts)
+        return {k: list(v) for k, v in jail_nics.items()}
+
+    @private
+    def start_failover_checks(self, jail_config):
+        if not self.middleware.call_sync('system.is_freenas') and self.middleware.call_sync('failover.licensed'):
+            jail_config = {
+                k: ioc_common.check_truthy(v) if k in IOCJson.truthy_props else v for k, v in jail_config.items()
+            }
+            if jail_config.get('dhcp'):
+                jail_config['vnet'] = 1
+            if not (jail_config['vnet'] or jail_config['nat']):
+                return
+            failover_enabled = not self.middleware.call_sync('failover.config')['disabled']
+            to_disable_nics = self.middleware.call_sync('jail.nic_capability_checks', [jail_config])
+            if to_disable_nics:
+                if failover_enabled:
+                    raise CallError(
+                        f'Failover must be disabled before creating/updating {jail_config["id"]} '
+                        f'because of the following nics: {",".join(to_disable_nics)}'
+                    )
+                else:
+                    for nic in to_disable_nics:
+                        self.middleware.call_sync('interface.disable_capabilities', nic, to_disable_nics[nic])
+                        try:
+                            self.middleware.call_sync(
+                                'failover.call_remote', 'interface.disable_capabilities', [nic, to_disable_nics[nic]]
+                            )
+                        except Exception as e:
+                            self.middleware.logger.debug(
+                                f'Failed to disable capabilities for {nic} on remote node: {e}'
```

Comment:
The `remote node` was removed as accepted verbiage for TrueNAS HA. Let's use `for {nic} on standby storage controller`, please.